lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -365,6 +367,21 @@ def add_address(post, options)
         end
       end
 
+      def add_statement_address(post, options)
+        statement_address = options[:statement_address]
+
+        return unless statement_address && statement_address[:address1] &&
+          statement_address[:city] &&
+          statement_address[:zip] &&
+          statement_address[:state]
+        post[:statement_address] = {}
+        post[:statement_address][:line1] = statement_address[:address1]
+        post[:statement_address][:line2] = statement_address[:address2]
+        post[:statement_address][:city] = statement_address[:city]
+        post[:statement_address][:postal_code] = statement_address[:zip]
```

Comment:
Nope, you've got it right. ActiveMerchant's contact for `billing_address` and `shipping_address` is as follows: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L39-L49